library.html

```diff
@@ -35,155 +35,174 @@
 			</div>
 			
 			<!--Content Boxes-->
-				<div class = "content_3">
+			<div class = "main_content">
+				
+				<div class = "title2">
 
-					<heading1>
+						<b>
```

Comment:
This is the text for music display, it has margins and spacing as seen on the CSS style sheet.